accelerator/cpu_accelerator.py

```diff
@@ -0,0 +1,271 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import torch
+from deepspeed.accelerator.abstract_accelerator import DeepSpeedAccelerator
+import oneccl_bindings_for_pytorch  # noqa: F401
+import psutil
+
+
+# accelerator for Intel CPU
+class CPU_Accelerator(DeepSpeedAccelerator):
+
+    def __init__(self):
+        self._name = 'cpu'
+        self._communication_backend_name = 'ccl'
+        self.max_mem = psutil.Process().memory_info().rss
+
+        def _check_and_mapping_mpich_env():
+            # Workaround when using Intel MPI launcher
+            # Model can check "RANK" and "WORLD_SIZE" for rank and world size
+            import os
+            if "RANK" not in os.environ and "PMI_RANK" in os.environ:
+                os.environ['RANK'] = os.environ.get('PMI_RANK')
+                print("mapping environment variable PMI_RANK to RANK")
```

Comment:
To reduce log size, can you please change the 3 prints as follows:
1. Restrict to only rank 0
2. Use [log_dist](https://github.com/microsoft/DeepSpeed/blob/085981bf1caf5d7d0b26d05f7c7e9487e1b35190/deepspeed/utils/logging.py#L75).
3. Combine into single print statement

Reply:
Thanks for the suggestion.  We plan to follow this PR https://github.com/microsoft/DeepSpeed/pull/3372 to move environment setting into multinode runner, rather than check and map in accelerator init function.  After that, we will remove the mapping and logging code in this PR.